### DISCUSSION TITLE
Requires confirmation on Save Edit Modal

### DISCUSSION THREAD

**Discussion by guuhgalvao** [filamentphp/filament#6933] [Help]
Hi guys, 

I need help to show a confirmation modal on click in the save button of edit modal (i'm using simple modal resource).

Basically, showing a modal inside another modal.

Can someone help me?

**Comment by pboivin** [comment]
Hi @guuhgalvao, I'm not sure this is possible... You may be able to replace the 'Save' action of the modal with a custom action, and add a call to JS `confirm()` : 
```
    ->extraAttributes(['onclick' => new HtmlString("return confirm('Are you sure you want to save?')")])
```
This is a bit hacky but maybe it can give you another idea.

**Reply by guuhgalvao** [reply]
Hi @pboivin, thanks for sharing this idea, it might help.

**Reply by parrotsoft** [reply]
Hello, thank you very much, question: if the confirm dialog is confirmed, how do I trigger the action to delete the record?

**Reply by parrotsoft** [reply]
It this 
```php
Action::make('delete')
    ->label(trans('generals.buttons.delete'))
    ->icon('heroicon-o-trash')
    ->color('danger')
    ->extraAttributes(fn (CustomExport $record) => ['onclick' => new HtmlString("
        if(confirm('Are you sure you want to delete record?')) {
           Livewire.emit('delete-record', {$record->id});
        }
    ")]);
